frontend/app/components/ToolCard.tsx

```diff
@@ -30,24 +27,7 @@ export default function ToolCard (props: ToolCardProps) {
   const { tool } = props;
 
   const [showMoreInfo, setShowMoreInfo] = useState(false);
-
-  let logoImageProps: CardImageProps = {
-    component: 'img',
-    height: 150,
-    alt: tool.logo_image_alt_text ?? 'Logo for tool'
```

Comment:
Can the alt text include tool name, such as "Logo image for Gradecraft tool"?

Reply:
Sure

Reply:
Should be changed now.

Reply:
verified!